Untitled.ipynb

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
As a matter of fact, ipynb files aren't really easy on the eye, typically jupyther notebooks aren't really project files, they are more like playground or sandbox files to try and figure things out.

But we can leave it here for now